tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class desiredCapabilitieTest extends TestCase
+{
+
+    public function testGetDesiredCapabilties()
+    {
+        $driver = $this->getSession()->getDriver();
+        $this->assertNotEmpty($driver->getDesiredCapabilities(), 'desiredCapabilities empty');
```

Comment:
Nobody prevents you from creating test case that doesn't use session at all and manually instantiate driver class.